src/univalue/univalue.h

```diff
@@ -154,7 +154,8 @@ class UniValue {
     bool push_back(std::pair<std::string,UniValue> pear) {
         return pushKV(pear.first, pear.second);
     }
-    friend const UniValue& find_value( const UniValue& obj, const std::string& name);
+    //!return a value if key was found (only supports VOBJ types)
```

Comment:
I feel like the appropriate comment would be `only useful for VOBJ types`,  as technically the function is still "supported".
A nit.